### PR TITLE
Bwduncan python3 fixes (fixed conflicts)

### PR DESCRIPTION
**Pull request by TrystanLea**

### CODE REVIEW

File: src/interfacers/tmp/EmonHubSmilicsInterfacer.py [hunk 1]
```diff
@@ -120,13 +116,13 @@ def _process_rx(self, smilics_dict):
...
             return c
-        except:
+        except Exception:
             return None
 
     def set(self, **kwargs):
         """ Override default settings with settings entered in the config file
         """
-        for key, setting in self._settings.iteritems():
+        for key, setting in self._settings.tems():
```
Comment (bwduncan) [inline src/interfacers/tmp/EmonHubSmilicsInterfacer.py:125]: This looks like a typo, should be `items()`.

**Comment by borpin** [comment]
Looking good. Can it stay in master for a while without merging to stable so it can be tested please?

File: src/interfacers/tmp/EmonHubSmilicsInterfacer.py (hunk 1 above)
Comment (TrystanLea) [reply src/interfacers/tmp/EmonHubSmilicsInterfacer.py:125]: Cheers @bwduncan fixed https://github.com/openenergymonitor/emonhub/commit/1e09d5a7b77aa820ba1dea8f2c7031916b277dd3

**Comment by TrystanLea** [comment]
@borpin sure, I've merged to master for further testing

**Pull request merged**